.circleci/config.yml

```diff
@@ -44,6 +44,41 @@ jobs:
           name: Deploy Master to Heroku
           command: |
             git push --force git@heroku.com:$HEROKU_APP_NAME.git HEAD:refs/heads/master
+  
+  docs-build:
+    docker:
+      - image: circleci/node:11.4
+    steps:
+      - checkout
+      - run: 
+          name: Install NPM
+          command: npm install
+      - run:
+          name: Build Server Documentation
+          command: npm run gen-server-docs
+      - persist_to_workspace:
+          root: ~/
+          paths: ./project/docs/server
```

Comment:
What does this step do?

Reply:
Makes the docs/ directory "persisted" between runs of jobs - we need the `docs/` dir generated from `docs-build` in `docs-deploy`